src/ThreadedFFI-UFFI-Overrides/EditorFindReplaceDialogWindow.class.st

```diff
@@ -16,10 +16,10 @@ Class {
 	#classInstVars : [
 		'defaultFindReplaceServiceClass'
 	],
-	#category : #'Text-Edition-UI'
+	#category : #'ThreadedFFI-UFFI-Overrides'
 }
```

Comment:
this is odd... why move the class to ThreadedFFI-UFFI-Overrides?

Reply:
OK - created a new PR https://github.com/pharo-project/pharo/pull/12160